consul/consul/spec.yml

```diff
@@ -2,7 +2,7 @@
 name: consul
 version: 0.6.4
```

Comment:
I'm confused about the versions. If I run, `consul version`, I see

```
Consul 'v0.6.4-234-gbc6338d'.dev ('bc6338db0fdd7712943f5267ca71529518a883a7+CHANGES')
```

But, I see `0.7.0dev`

```
Node                         Address          Status  Type    Build     Protocol  DC
resching-aws-control-01      10.1.1.104:8301  alive   server  0.7.0dev  2         aws-us-east-1
...
```

Is this something mustwin needs to fix and should we sync the version in the hammer spec with the version of their binary?